nidmviewer/utils.py

```diff
@@ -55,7 +55,7 @@ def read_file_lines(file_name):
 
 def download_file(src,dest):
     try:
-        requester = urllib.URLopener()
+        requester = urllib.FancyURLopener()
         requester.retrieve(src, dest)
```

Comment:
woohoo! It looks like this will follow redirects (and general 3xx) responses up to 10 times, and then return the actual URL: https://docs.python.org/2/library/urllib.html#urllib.FancyURLopener. It looks like for python 3 it was moved under urllib.request, should we add a check and import based on what is found to support both 2 and 3? https://docs.python.org/3/library/urllib.request.html?highlight=fancyurl#urllib.request.FancyURLopener

Reply:
should support both now.